docs/release-notes/8.4.md

```diff
@@ -0,0 +1,155 @@
+  date and time inside the Linux containers.
+- EuroLinux 8 kernel now supports the Error Detection and Correction (EDAC) in
+  8th and 9th generation Intel Core Processors.
+- There are changes in the pacemaker resource agent - it now can detect
+  failures asynchronously and inject a failure into Pacemaker
+  immediately without waiting for the following monitor interval.
+
+## How to update from beta
+
+The beta repository has an updated package `el-release`, that contains production
+repositories. Upgrading from EuroLinux 8.4 beta to 8.4 is as simple as running:
+
+```
+sudo yum update -y el-release
+sudo yum update -y
+```
+
+## Branded packages
+
+Multiple packages have names changed due to branding, including:
+
+- redhat-backgrounds -> el-backgrounds
+- redhat-indexhtml -> el-indexhtml
+- redhat-logos -> el-logos
+- redhat-logos-httpd -> el-logos-httpd
+- redhat-logos-ipa -> el-logos-ipa
+- redhat-release -> el-release
+- redhat-release-eula -> el-release - we include EULA in release package
+
+All these packages have proper RPM `provides` declaration.
+
+The list of the packages that are changed due to branding includes, among the
+others:
+
+- abrt
+- anaconda
+- anaconda-user-help
+- anacondafirefox
+- cloud-init
+- crash
+- httpd
+- httpd
+- initial-setup
+- ipa
+- liberport
+- libguestfs
+- libreoffice
+- nginx
+- opa-ff
+- opa-fm
+- openscap
+- osindo-db
+- python3
+- redhat-rpm-config
+- scap-security-guide
+- skopeo
+- sos
+- subscription-manager
+- thunderbird
+
+
+## Batteries
+
+Batteries repository contains packages that are needed to rebuild distribution
+but are not included in upstream repositories. Right now, this repository
+contains more than 2000 packages for each buildroot.
+
+Multiple batteries were updated for this release there including:
+
+- mdds
+- meson
+- spirv-headers
+- fontforge
+- multiple python packages
+- modules for bootstrapping python39
+
+## Other notable changes
+
+- Because we changed the business model to open core, there is no longer a need
+  to use FBI (free base image) repositories as they are a subset of entire
+  repositories.  FBI repos are removed from the `el-release` package.
+- EuroLinux 8.4 is first version that can use `BaseOS-all`, `AppStream-all` and
+  `PowerTools-all` repositories, that contains **all** packages produced during
+  build process. However, this packages are not supported by upstream or
+  EuroLinux.
+- EuroLinux, by default, uses DockerHub instead of the Quay.
+
+## Additional resources
+
+- [Download EuroLinux ISO](https://fbi.cdn.euro-linux.com/isos/)
+- [EuroLinux Public Request for Change and Bug Tracker](https://fbi.cdn.euro-linux.com/isos/)
+- Roadmap for the project can be found in press notes available on our company
+  blog. [Press Notes](https://en.euro-linux.com/blog/).
+- [Red Hat 8.4
+  Release-Notes](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.4_release_notes/index)
```

Comment:
niepotrzebny myślnik w _Release-Notes_, można za to dać jednego po odnośniku, żeby linijka niżej dobrze się zgrała